src/kafka-util/src/bin/kgen.rs

```diff
@@ -0,0 +1,128 @@
+// Copyright Materialize, Inc. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use anyhow::bail;
+use clap::{App, Arg};
+use rand::{distributions::Uniform, prelude::Distribution, thread_rng, Rng};
+use rdkafka::{
+    error::KafkaError,
+    producer::{BaseRecord, DefaultProducerContext, ThreadedProducer},
+    types::RDKafkaError,
+    ClientConfig,
+};
```

Comment:
we don't enforce this (because it's impossible to) so TIOLI, but we do normally ensure that `use` lines have the crate at the beginning of all imports. 

Considering the state of rust tooling overall I like being able to `rg rdkafka.*Client` and have that be guaranteed to find all modules that use `rdkafka::ClientConfig`.